provider/datagen/src/bin/datagen/args.rs

```diff
@@ -232,8 +239,17 @@ pub struct Cli {
 
     // TODO(#2856): Change the default to Auto in 2.0
     #[arg(short, long, value_enum, default_value_t = Fallback::Hybrid)]
+    #[arg(help = "The fallback mode to use in the generated data. \
+                For details, read the Rust docs for FallbackMode.")]
     fallback: Fallback,
 
+    #[arg(long, value_enum, default_value_t = BaseLanguageHandling::Auto)]
+    #[arg(
+        help = "--fallback=runtime[-manual] only: Whether to retain or strip base languages. \
+                For details, read the Rust docs for BaseLanguageHandling."
```

Comment:
nit: let's call the local one BaseLanguageHandlingArg, to avoid confusion when looking at the code. The one above has no docs.

Alternatively give it docs saying to look at the type in the config module.